src/passes/wasm-intrinsics.wat

```diff
@@ -11,6 +11,8 @@
 ;;    special scratch space, wasm2js_scratch_load_i32 etc.
 ;;  * Fix function type of __wasm_ctz_i64, which was wrong somehow,
 ;;    i32, i32 => i32 instead of i64 => i64
+;;  * Add $wasm-intrinsics-temp-i64 global to avoid a 64-bit import
+;;    wasm2js_scratch_load_i64/store
```

Comment:
IIUC, this name won't exist in the codebase after this change, so it would probably be good to avoid using it in this comment.

Reply:
Sure, updated.

I kind of saw the lines here as a "history", but yeah, maybe that's confusing.